data/mainnet/addresses.json

```diff
@@ -1,4 +1,5 @@
 {
+  "ibEUR": "0x96e61422b6a9ba0e068b6c5add4ffabc6a4aae27",
```

Comment:
Is this used?

Reply:
![image](https://user-images.githubusercontent.com/34766620/135623069-1028fa8b-e29a-4fbd-84c4-9695a5eacb6f.png)
yes this is used.
although i'm not sure if I defined the priceFunction correctly

Reply:
Ah, right. Yeah looks good